client/src/app/site/pages/meetings/modules/projector/modules/slides/components/motions/modules/motion-slide/components/motion-slide/motion-slide.component.scss

```diff
@@ -79,7 +79,7 @@
     position: relative;
     left: -40px;
     height: 1.5em;
-    margin-top: 25px;
+    margin-top: 15px;
```

Comment:
please remove `margin-top` and `left`. Label should placed within the text block/paragraph.